extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetColumnLocation.java

```diff
@@ -684,9 +686,7 @@ public Object getGrouping() {
                 return Optional
                         .of(ToDateTimePage.create(componentType, timestampLogicalType.getUnit()));
             }
-
-            throw new TableDataException(
```

Comment:
We decided a while back that a better strategy from a usability perspective is to load untyped instead of completely failing to load.

Reply:
Yes, we decided a while back that as a general way to handle logical types that we don't support, that beats blowing up loading anything with an error.